C-Plus-Plus/other/merge_2_sorted_arrays_without_extra_space.cpp

```diff
@@ -0,0 +1,57 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main() {
+
+  int m, n;
+  cout << "Enter the total number of elements in array1 and array2 respectively\n";
+  cin >> m >> n;
+
+  int arr1[m], arr2[n];
+  cout << "enter 1st array elements\n";
+  for (int i = 0; i < m; i++) {
+    cin >> arr1[i];
+  }
+  cout << "enter 2nd array elements\n";
+  for (int i = 0; i < n; i++) {
+
+    cin >> arr2[i];
+  }
+
+  for (int i = m - 1, j = 0; i >= 0 && j < n; i--, j++) {
```

Comment:
Use proper comments to explain your code.

Reply:
Done @atarax665